marlowe-playground-client/src/Simulation.purs

```diff
@@ -648,7 +649,9 @@ inputItem state isEnabled person (MoveToSlot slot) =
     else
       []
 
-  inFuture = maybe false (\x -> x.slot < slot) (state ^. (_currentMarloweState <<< _executionState))
+  mCurrentSlot = preview (_currentMarloweState <<< _executionState <<< _Just <<< _slot) state
+
+  inFuture = maybe false ((>) slot) mCurrentSlot
```

Comment:
can use `is` and https://pursuit.purescript.org/packages/purescript-profunctor-lenses/6.2.0/docs/Data.Lens.Prism#v:nearly here